Podfile

```diff
@@ -26,15 +26,14 @@ target 'CodableContextTests' do
 end
 
 target 'Sourcery' do
-  pod 'Stencil', '0.10.1'
-  pod 'StencilSwiftKit', '2.4.0' 
-  pod 'Commander', '0.6.0'
-  pod 'PathKit', '0.8.0'
-  pod "xcproj", '4.2.0'
-  pod 'SourceKittenFramework', '0.21.1'
+  pod 'Stencil', '0.12.1'
+  pod 'StencilSwiftKit', '2.6.0'
+  pod 'Commander', '0.7.0'
+  pod 'PathKit', '0.9.2'
+  pod "xcproj", :git =>'git@github.com:tuist/xcodeproj.git', :tag => '4.3.1'
```

Comment:
What is the reason you use a fork instead of `https://github.com/xcodeswift/xcproj.git` here? Because you're using the main repo in the [SPM manifest](https://github.com/krzysztofzablocki/Sourcery/pull/683/files#diff-37ca2dd15ca0f6b1b49e78db084ef5b9R20) 🤔

Reply:
`xcodeswift/xcproj.git` redirects to `tuis/xcodeproj.git`

Reply:
it's a shame that they didn't release the 4.3.1 version properly with cocoapods..